packages/wonder-blocks-button/components/button.js

```diff
@@ -102,6 +101,8 @@ export type SharedProps = {|
     /**
      * Optional custom styles.
      */
+    innerStyle?: StyleType,
+
```

Comment:
Could you add a comment explaining the difference between `innerStyle` and `style`? I think adding a comment to `style` would help.

Reply:
yes will do, I had to implement this prop due to the new View which wraps the buttons contents

Reply:
Agree that comments will be super helpful.  It might be worth adding a small diagram showing the nesting of elements showing which styles are being applied where.